### ISSUE TITLE
1.55d Bug bonus skills from set bonus

### ISSUE THREAD

**Issue by DimasjkTV** [suriyun-production/mmorpg-kit-docs#474]
If we have a set that gives a bonus to a skill, then this skill moves to the top of the list. This disrupts the skill tree sequence.
[1.55d Bug bonus skills from set bonus.zip](https://github.com/suriyun-production/mmorpg-kit-docs/files/4756263/1.55d.Bug.bonus.skills.from.set.bonus.zip)

**Comment by insthync** [comment]
You can fix it by move codes 
```
        // Prepare equipment set bonus
        data.GetEquipmentSetBonus(ref resultStats, resultAttributes, resultResistances, resultArmors, resultIncreaseDamages, resultSkills, resultEquipmentSets, true);
```
in `CharacterDataExtension` function `GetAllStats` from line `1580` to `1587`

![Screenshot_203](https://user-images.githubusercontent.com/2697550/84559326-d54a3800-ad63-11ea-98e1-f6f9bc75e9d1.png)

**Comment by insthync** [comment]
Fixed in 1.56
